src/helpers/host-component-names.tsx

```diff
@@ -29,12 +30,20 @@ export function configureHostComponentNamesIfNeeded() {
 
 function detectHostComponentNames(): HostComponentNames {
   try {
-    const renderer = TestRenderer.create(
-      <View>
-        <Text testID="text">Hello</Text>
-        <TextInput testID="textInput" />
-      </View>
-    );
+    const renderer = (() => {
+      let result: ReactTestRenderer;
+      TestRenderer.act(() => {
+        result = TestRenderer.create(
+          <View>
+            <Text testID="text">Hello</Text>
+            <TextInput testID="textInput" />
+          </View>
+        );
+      });
+
+      // @ts-ignore act is syncronous, so renderer will already be initialised here
```

Comment:
There's a typo, missing 'h' in 'synchronous'

Reply:
Spotted it and fixed! :-)